packages/@expo/fingerprint/src/sourcer/Bare.ts

```diff
@@ -83,12 +83,16 @@ export async function getRncliAutolinkingSourcesAsync(
     const reasons = ['bareRncliAutolinking'];
     const autolinkingConfig: Record<string, any> = {};
     for (const [depName, depData] of Object.entries<any>(config.dependencies)) {
-      stripRncliAutolinkingAbsolutePaths(depData, root);
-      const filePath = depData.root;
-      debug(`Adding react-native-cli autolinking dir - ${chalk.dim(filePath)}`);
-      results.push({ type: 'dir', filePath, reasons });
+      try {
```

Comment:
Added an extra try/catch so that only the dependencies that have an issue are not included in the results